Ktisis/Locale/i18n/fr_FR.json

```diff
@@ -0,0 +1,407 @@
+{
+	"$meta": {
+		"displayName": "French",
+		"selfName": "Français",
+		"__comment": "Using null element to signal 'et al' in the array below. Don't do this for your own localization.",
+		"maintainers": [
+			"@Luna664",
+			null
```

Comment:
As the comment above states, "Don't [use a null element] for your own localization".